packages/components/src/components/notifications/Toast/useToast.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { screen } from '@tests/renderWithProviders';
+import { renderHook, act } from '@testing-library/react-hooks';
```

Comment:
Do we want to export this library inside of @tests? @tkanzakic

Reply:
yes, IMO it make sense to have a custom `renderHook` that already wraps the component in a `ComposerProvider`